Lib/fontbakery/codetesting.py

```diff
@@ -147,16 +147,21 @@ def assert_results_contain(check_results, expected_status, expected_msgcode=None
     order among other log messages.
     """
     from fontbakery.message import Message
+    from fontbakery.checkrunner import PASS, DEBUG
     if not reason:
         reason = f"[{expected_msgcode}]"
+    if not expected_msgcode:
+      raise Exception("Test must expect a message code")
```

Comment:
check! ;-)

Reply:
No, test! ;-) This part is testing the integrity of the test suite. When calling ` assert_results_contain` the test should expect a message code.